### PR TITLE
Update httpclient to 4.5.13.wso2v1

### PR DESCRIPTION
**Pull request by sadilchamishka**
### Proposed changes in this pull request
Update httpclient version from 4.3.6.wso2v1 to 4.5.13.wso2v1

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -920,7 +920,7 @@
         <spring-web.version>4.3.29.RELEASE</spring-web.version>
         <swagger-jaxrs.version>1.6.2</swagger-jaxrs.version>
         <!-- httpclient dependency Version-->
-        <http.client.version>4.3.6.wso2v1</http.client.version>
+        <http.client.version>4.5.13.wso2v1</http.client.version>
```
Comment (nilasini) [inline pom.xml:923]: don't we need to update the http core version?

Comment (sadilchamishka) [reply pom.xml:923]: Will update the http core to the latest version

**Review by nilasini — APPROVED** [review]

File: pom.xml (hunk 1 above)
Comment (dewniMW) [reply pom.xml:923]: is this addressed?

Comment (sadilchamishka) [reply pom.xml:923]: yes,  https://github.com/wso2-extensions/identity-inbound-auth-oauth/pull/1830/files#diff-9c5fb3d1b7e3b0f54bc5c4182965c4fe1f9023d449017cece3005d3f90e8e4d8R924

**Pull request merged**